app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -1,4 +1,6 @@
 ## Override from hyrax to fix file upload in logo and banner
+## OVERRIDE: Hyrax 2.9 to use work titles for collection thumbnail select & to add an option to reset to the default thumbnail
```

Comment:
in the comments below, can you explicitly state "OVERRIDE: ...." for the overridden code that correlates to this comment?